vispy/visuals/tests/test_volume.py

```diff
@@ -29,7 +29,7 @@ def test_volume():
 
     # Clim
     V.set_data(vol, (0.5, 0.8))
-    assert V.clim == (np.float32(0.5), np.float32(0.8))
+    assert V.clim == (0.5, 0.8)
```

Comment:
At least locally, this failed for me and is why I changed it...I think. If it works now with your changes then great.